.gitignore

```diff
@@ -1 +1,3 @@
 target/
+.idea/
+.bsp/
```

Comment:
Can these be added either in a specific commit (with a clean commit history overall), or in a separate PR?